### ISSUE TITLE
Self-update notification webview is blank

### ISSUE THREAD

**Issue by kakra** [ValveSoftware/steam-for-linux#7770]
#### Your system information

* Steam client version (build number or date): beta as of 2021-04-27
* Distribution (e.g. Ubuntu): Gentoo
* Opted into Steam client beta?: [Yes/No] Yes
* Have you checked for system updates?: [Yes/No] Yes

#### Please describe your issue in as much detail as possible:

Since some weeks now, whenever the notification window for self-update displays, the webview containing the changes is completely blank:
![image](https://user-images.githubusercontent.com/17684/116289609-eb51c300-a792-11eb-8863-337e1cced285.png)

But the contents seem to be there: The moment I click "Restart" (Neu starten), the contents instantly display but almost the very same moment the window would close to restart the client. This makes it pretty hard to follow any update changelogs.

#### Steps for reproducing this issue:

1. Wait for the self-update notification

[Steam system info](https://gist.github.com/kakra/a92cdfc0ed8cc212ae981a7d9518b5a6)

**Comment by kisak-valve** [comment]
Hello @kakra, this issue is already being tracked at #6451. Closing as a duplicate.

**Comment by kakra** [comment]
Thanks, issue search didn't reveal that to me.
